met-api/tests/unit/api/test_engagement_membership.py

```diff
@@ -5,118 +5,116 @@
-def test_create_engagement_membership_reviewer(mocker, client, jwt, session,
-                                               setup_admin_user_and_claims):
-    """Assert that a reviewer engagement membership can be created."""
-    user, claims = setup_admin_user_and_claims
-    engagement = factory_engagement_model()
-    staff_user = factory_staff_user_model()
-    headers = factory_auth_header(jwt=jwt, claims=claims)
-
-    mock_assign_composite_role_to_user_keycloak_response = MagicMock()
-    mock_assign_composite_role_to_user_keycloak_response.status_code = HTTPStatus.NO_CONTENT
-    mock_assign_composite_role_to_user_keycloak = mocker.patch(
-        'met_api.services.keycloak.KeycloakService.assign_composite_role_to_user',
-        return_value=mock_assign_composite_role_to_user_keycloak_response
-    )
-    mock_get_users_roles_keycloak = mocker.patch(
-        'met_api.services.keycloak.KeycloakService.get_users_roles',
-        return_value={staff_user.external_id: [KeycloakCompositeRoleNames.REVIEWER.value]}
-    )
-
-    data = {'user_id': staff_user.external_id}
-
-    rv = client.post(
-        memberships_url.format(engagement.id),
-        data=json.dumps(data),
-        headers=headers,
-        content_type=ContentType.JSON.value
-    )
-    assert rv.status_code == HTTPStatus.OK
-    assert rv.json.get('engagement_id') == engagement.id
-    assert rv.json.get('user_id') == staff_user.id
-    assert rv.json.get('type') == MembershipType.REVIEWER
-    assert rv.json.get('status') == MembershipStatus.ACTIVE.value
-    mock_assign_composite_role_to_user_keycloak.assert_called()
-    mock_get_users_roles_keycloak.assert_called()
-
-
-def test_create_engagement_membership_unauthorized(client, jwt, session,
-                                                   setup_unprivileged_user_and_claims):
-    """Assert that creating an engagement membership without proper authorization fails."""
-    user, claims = setup_unprivileged_user_and_claims
-    engagement = factory_engagement_model()
-    staff_user = factory_staff_user_model()
-    headers = factory_auth_header(jwt=jwt, claims=claims)
-    data = {'user_id': staff_user.external_id}
-
-    rv = client.post(
-        memberships_url.format(engagement.id),
-        data=json.dumps(data),
-        headers=headers,
-        content_type=ContentType.JSON.value
-    )
-    assert rv.status_code == HTTPStatus.FORBIDDEN
+# TODO: Replace this test with one that adds composite roles to user
+# def test_create_engagement_membership_team_member(mocker, client, jwt, session,
+#                                                   setup_admin_user_and_claims):
+#     """Assert that a team member engagement membership can be created."""
+#     user, claims = setup_admin_user_and_claims
+#     engagement = factory_engagement_model()
+#     staff_user = factory_staff_user_model()
+#     headers = factory_auth_header(jwt=jwt, claims=claims)
+
+#     mock_add_user_to_group_keycloak_response = MagicMock()
```

Comment:
Of course these are commented out and will be restored later but just make sure not to miss changing "group" in the future.